packages/node/src/dev-server.ts

```diff
@@ -82,7 +92,141 @@ function listen(server: Server, port: number, host: string): Promise<void> {
   });
 }
 
-let bridge: Bridge | undefined = undefined;
+async function createServerlessEventHandler(
```

Comment:
This is an extraction of the existing logic. See the individual commit for clarity: [`6b9d5fd` (#7925)](https://github.com/vercel/vercel/pull/7925/commits/6b9d5fd4c832d1e31267d2f8d4094ff154fc9ccc)